core/rawdb/schema.go

```diff
@@ -387,3 +390,58 @@ func utxoToBlockHeightKey(txHash common.Hash, index uint16) []byte {
 	txHash[common.HashLength-2] = indexBytes[1]
 	return append(utxoToBlockHeightPrefix, txHash[:]...)
 }
+
+var CoinbaseLockupKeyLength = len(CoinbaseLockupPrefix) + 2*common.AddressLength + 1 + 4
+
+func CoinbaseLockupKey(ownerContract common.Address, beneficiaryMiner common.Address, lockupByte byte, epoch uint32) []byte {
```

Comment:
add couple test cases, would be nice

Reply:
Make an issue and put it in my todo

Reply:
https://github.com/dominant-strategies/go-quai/issues/2462